### PR TITLE
Documentation of hyperlink_label

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Documentation of the function hyperlink_label. Description and inputs in need of completion

### CODE REVIEW

File: ea_hyperlink_label.m [hunk 1]
```diff
@@ -1,4 +1,20 @@
 function ea_hyperlink_label(label, url, position)
+%
```
Comment (Daniel-Tojal) [inline ea_hyperlink_label.m]: Please provide a description of the function

File: ea_hyperlink_label.m [hunk 2]
```diff
@@ -1,4 +1,20 @@
 function ea_hyperlink_label(label, url, position)
+% 
+%
+% USAGE:
+%
+%    ea_hyperlink_label(label, url, position)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_hyperlink_label.m]: Please provide a description of the inputs

File: ea_hyperlink_label.m (hunk 1 above)
Comment (ningfei) [reply ea_hyperlink_label.m]:
```
%  Create a clickable hyperlink label
%
% INPUTS:
%    label: label text
%    url: linked URL
%    position: position of the label
```

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal and @ningfei 👍

**Pull request merged**